node/src/pruner.rs

```diff
@@ -360,7 +360,8 @@ where
         // <-----------Non sampling area----------->|<--Sampling window->
         //
         // This case is straight forward: we just remove whatever is after
```

Comment:
```suggestion
        // This case is straightforward: we just remove whatever is after
```
If we're already here, then I guess ;p